### PR TITLE
Rewrite scribblehub.com with browser

### PR DESCRIPTION
**Pull request by zerty**
issues: #1942 #1896 #1788

### CODE REVIEW

File: sources/en/s/scribblehub.py [hunk 1]
```diff
@@ -41,92 +51,79 @@ def initialize(self) -> None:
...
-        table td {
-            padding: 10px;
-        }
-        </style>
-        """
-        return body
+    def parse_chapter_list(
+        self, soup: BeautifulSoup
+    ) -> Generator[Union[Chapter, Volume], None, None]:
+        pass
```
Comment (dipu-bd) [inline sources/en/s/scribblehub.py:122]: missing implementation

Comment (zerty) [reply sources/en/s/scribblehub.py]: done

File: sources/en/s/scribblehub.py [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
 
+digit_regex = re.compile(r"\?toc=(\d+)#content1$")
 
-class ScribbleHubCrawler(Crawler):
+
+class ScribbleHubCrawler(SearchableBrowserTemplate):
     base_url = [
         "https://www.scribblehub.com/",
         "https://scribblehub.com/",
     ]
```
Comment (Seven0492) [inline sources/en/s/scribblehub.py:25]: Same thing here, it'd be nice if it could also support `http`.

Comment (kuwoyuki) [reply sources/en/s/scribblehub.py:25]: What's the use case for this? An end-user will never paste an `http` scheme URL on a website that has an `https` redirect scheme.

Comment (zerty) [reply sources/en/s/scribblehub.py:25]: let's stay in https

**Pull request merged**